### ISSUE TITLE
RDP plugin missing in version 1.4.33

### ISSUE THREAD

**Issue by StayPirate** [flathub/org.remmina.Remmina#221]
This seems to be a reoccurring problem (https://github.com/flathub/org.remmina.Remmina/issues/47, https://github.com/flathub/org.remmina.Remmina/issues/37, https://github.com/flathub/org.remmina.Remmina/issues/11) that @antenore was always able to quickly address. Might I ask you if the root cause is always the same? If yes, could you share some light on it?

```bash
❯ flatpak info org.remmina.Remmina

Remmina - Remote Desktop Client

          ID: org.remmina.Remmina
         Ref: app/org.remmina.Remmina/x86_64/stable
        Arch: x86_64
      Branch: stable
     Version: 1.4.33
     License: GPL-2.0+
      Origin: flathub
  Collection: org.flathub.Stable
Installation: user
   Installed: 17.7 MB
     Runtime: org.gnome.Platform/x86_64/44
         Sdk: org.gnome.Sdk/x86_64/44

      Commit: 8c434b2679b744edb511699bc5649c71fdf993f94613e947cb9fdd1a91080eba
      Parent: f99337fc310bc25e60e2f88fa10a4ec31e29f2f4e484b048a50434c9137d0193
     Subject: Move to 44 (88375828)
        Date: 2024-01-09 15:34:13 +0000
```

```bash
❯ flatpak --user run --filesystem=$SSH_AUTH_SOCK --env=SSH_AUTH_SOCK=$SSH_AUTH_SOCK org.remmina.Remmina
** Message: 09:50:15.082: Remmina does not log all output statements. Turn on more verbose output by using "G_MESSAGES_DEBUG=all" as an environment variable.
More info available on the Remmina wiki at:
https://gitlab.com/Remmina/Remmina/-/wikis/Usage/Remmina-debugging
Load modules from /app/lib/remmina/plugins
xprop:  unable to open display ''
xprop:  unable to open display ''
Remmina plugin glibsecret (type=Secret) has been registered, but is not yet initialized/activated. The initialization order is 2000.
The glibsecret secret plugin has been initialized and it will be your default secret plugin
```

**Comment by StayPirate** [comment]
From the FreeRDP upstream I see they have a stable 3x branch (currently [v3.1.0](https://github.com/FreeRDP/FreeRDP/releases/tag/3.1.0)). Here we are using [v2.11.2](https://github.com/flathub/org.remmina.Remmina/blob/8837582892b4fd87e38aef8647c6d49b04652c46/org.remmina.Remmina.json#L499-L500), do you think we could try to make a version bump to the most recent branch? Or maybe you already tested it before and didn't work?

**Comment by myheroyuki** [comment]
Hi @StayPirate, I am not sure what the root cause of this could be. I just tried running the same version of the flatpak as you (version 1.4.33, commit 8c434b2679b744edb511699bc5649c71fdf993f94613e947cb9fdd1a91080eba) and the RDP plugin is present and works to establish RDP connections. Have you checked if you have another instance of Remmina running? That could interfere with the flatpak. 

As for the stable 3x branch, we cannot just update to it in the flatpak as FreeRDP's API changes between 2x and 3x. There is currently a merge request open on our Gitlab that updates Remmina to work with either version of FreeRDP, so once that's merged in we will be able to update to 3.1.

**Comment by StayPirate** [comment]
You are right, after I deleted these two following folders then flatpaked Remmina started to work.

```bash
❯ rm -r ~/.config/remmina
❯ rm -r ~/.local/share/remmina
```

Thank you, closing.
